### ISSUE TITLE
Wondering isit possible to run this model with the description that I want?

### ISSUE THREAD

**Issue by KevinToh0329** [hanzhanggit/StackGAN#50]
I am wondering isit possible to run this model with the description that I want? 
If yes, can teach me how?

**Comment by akhilvasvani** [comment]
Yes, you can. Use the skip-thoughts model. It will convert your own sentences into word-embeddings to be trained on the StackGAN model
